lib/gui/css/main.css

```diff
@@ -6402,30 +6402,30 @@ body {
 .component-drive-selector-body .list-group-item-text {
   word-break: break-all; }
 
-.modal-drive-selector-modal .list-group {
+.component-drive-selector-body .list-group {
```

Comment:
Where are these CSS changes coming from? Looks like this is accidentally including unrelated changes.